aiAssistant.ts

```diff
@@ -6,17 +6,18 @@ import { functions } from './functions'
 const openai = new OpenAI()
 
 // Run this only once to create the assistant!
+// npm run create-ai-assistant
 export async function createAIDeveloper() {
   const aiDeveloper = await openai.beta.assistants.create({
     instructions: `You are an AI developer.
     When given a coding task, write and save code to files, install any packages if needed, make commits, and finally create a PR once done. You're logged in using GitHub CLI and have all the needed credentials.
     Start by listing all files inside the repo. You work inside the '/home/user/repo' directory where the repository is already cloned so you don't need to clone it yourself.
-    Don't argue with me and just complete the task.`,
+    Don't argue with me and just complete the task. You also have the ability to actually run the code or commands.`,
```

Comment:
You probably want something like this
```suggestion
    You also have the ability to actually run the Python code or Shell commands.
    Don't argue with me and just complete the task.`,
```